.github/workflows/android-e2e-test-fabric.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           disable-animations: false
           force-avd-creation: false
           emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          avd-name: e2e_emulator
+          rns_e2e_avd_name: e2e_emulator
```

Comment:
`avd-name` is a configuration option from the [github action this job step uses](https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#configurations). Renaming here does not make much sense. I think you went too far with find & replace 😅